### PR TITLE
docs: add example for: awscc_route53_hosted_zone

### PR DESCRIPTION
**Pull request by ishwar-amzn**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

Relates OR Closes #0000

### CODE REVIEW

File: examples/resources/awscc_route53_hosted_zone/awscc_route53_hosted_zone_private.tf [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    {
+      key   = "Name"
+      value = "this"
+    },
+    {
+      key   = "Modified By"
+      value = "AWSCC"
+    }
+  ]
+  vp_cs = [
```
Comment (wellsiau-aws) [inline examples/resources/awscc_route53_hosted_zone/awscc_route53_hosted_zone_private.tf:13]: this part doesnt read right, lets investigate this a bit

Comment (wellsiau-aws) [reply examples/resources/awscc_route53_hosted_zone/awscc_route53_hosted_zone_private.tf:13]: https://github.com/hashicorp/terraform-provider-awscc/blob/2e5e4089ce02498d5dbfb40d5e2900831b284f8a/internal/aws/route53/hosted_zone_resource_gen.go#L237C4-L237C9

Comment (wellsiau-aws) [reply examples/resources/awscc_route53_hosted_zone/awscc_route53_hosted_zone_private.tf:13]: @ewbankkit , is there a overrides that explain why this attribute map from "VPC" into "vp_cs"

**Review by wellsiau-aws — CHANGES_REQUESTED** [review]
please don't merge until we address the peculiarity with the `vp_cs` attribute

**Comment by marcosentino** [comment]
@ishwar-amzn Thank you for the contribution!

**Pull request merged**